protocols/tcp/tcp.go

```diff
@@ -28,6 +29,58 @@ type tcpServer struct {
 	events []parsedTCP
 }
 
+//go:embed responses/*
+var embeddedFiles embed.FS
+
+var responseFileMap = map[uint16]string{
+	21:    "responses/21_tcp",
+	25:    "responses/25_tcp",
+	80:    "responses/80_tcp",
+	110:   "responses/110_tcp",
+	135:   "responses/135_tcp",
+	139:   "responses/139_tcp",
+	445:   "responses/445_tcp",
+	1433:  "responses/1433_tcp",
+	3306:  "responses/3306_tcp",
+	4444:  "responses/4444_tcp",
+	4899:  "responses/4899_tcp",
+	5060:  "responses/5060_tcp",
+	5900:  "responses/5900_tcp",
+	8009:  "responses/8009_tcp",
+	21000: "responses/21000_tcp",
+}
+
+func (s *tcpServer) getResponse(port uint16) ([]byte, error) {
+	filePath, exists := responseFileMap[port]
+	if !exists {
+		return nil, fmt.Errorf("file not found for port: %v", port)
+	}
+
+	return embeddedFiles.ReadFile(filePath)
+}
+
+func (s *tcpServer) sendPortSpecificResponse(conn net.Conn, port uint16, logger interfaces.Logger) error {
```

Comment:
it is port-specific or random. I'd keep the "or random" part in the HandleTCP method.